rpcs3/Emu/Cell/Modules/cellSearch.cpp

```diff
@@ -42,31 +44,91 @@ void fmt_class_string<CellSearchError>::format(std::string& out, u64 arg)
 	});
 }
 
+enum class SearchState
```

Comment:
Fix style of the enum (no allcaps)

Reply:
`snake_case` or `TitleCase`? I see both are used in other enum classes.

Reply:
cell_search_state (snake case), `cell_` prefix indicates its exclusivity to cellSearch.
for values ss well, e.g. cell_search_state::inited etc.

Reply:
TitleCase is used in cell. So we use snake_case for our stuff (apart from GUI which is unrelated to emucore)